sql/core/pom.xml

```diff
@@ -153,7 +153,7 @@
     <dependency>
       <groupId>com.h2database</groupId>
       <artifactId>h2</artifactId>
-      <version>1.4.195</version>
+      <version>2.0.202</version>
```

Comment:
Shall we use `2.0.204`?
- https://repo1.maven.org/maven2/com/h2database/h2/2.0.204/

Reply:
@dongjoon-hyun OK